pkg/operator/configobservation/serviceca/observeserviceca.go

```diff
@@ -0,0 +1,68 @@
+package serviceca
+
+import (
+	"k8s.io/apimachinery/pkg/api/equality"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	"github.com/openshift/cluster-kube-controller-manager-operator/pkg/operator/configobservation"
+	"github.com/openshift/cluster-kube-controller-manager-operator/pkg/operator/operatorclient"
+	"github.com/openshift/library-go/pkg/operator/configobserver"
+	"github.com/openshift/library-go/pkg/operator/events"
+)
+
+const (
+	serviceCAFilePath      = "/etc/kubernetes/static-pod-resources/configmaps/service-ca/service-ca.crt"
+	serviceCAConfigMapName = "service-ca"
+)
+
+// ObserveServiceCA fills in serviceServingCert.CertFile with the path for a configMap containing the service-ca.crt
+func ObserveServiceCA(genericListers configobserver.Listers, recorder events.Recorder, existingConfig map[string]interface{}) (map[string]interface{}, []error) {
+	listers := genericListers.(configobservation.Listers)
+	errs := []error{}
+	prevObservedConfig := map[string]interface{}{}
+
+	topLevelServiceCAFilePath := []string{"serviceServingCert", "certFile"}
+	currentServiceCAFilePath, _, err := unstructured.NestedString(existingConfig, topLevelServiceCAFilePath...)
+	if err != nil {
+		errs = append(errs, err)
+	}
+	if len(currentServiceCAFilePath) > 0 {
+		if err := unstructured.SetNestedField(prevObservedConfig, currentServiceCAFilePath, topLevelServiceCAFilePath...); err != nil {
+			errs = append(errs, err)
+		}
+	}
+
+	// Sync the configMap in openshift-config-managed to the well-known resting place that corresponds to the service-ca path.
+	// If neither are set, this updates the destination with an empty source, which deletes the destination resource.
+	if err != nil {
```

Comment:
I think this got left in from an earlier commit.

Reply:
🤦‍♀️ a257842a